pom.xml

```diff
@@ -21,6 +21,10 @@
         <spring.version>5.3.3</spring.version>
     </properties>
     <dependencies>
-
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
     </dependencies>
 </project>
```

Comment:
missing empty line at the end of the file

Reply:
Thanx. added.